docs/docs/30-how-to-guides/14-working-with-stages.md

```diff
@@ -0,0 +1,421 @@
+
+For example:
+
+```yaml
+status:
+  freightHistory:
+  - id: 101bca5b0e18ca7913978a1da956308d2544f741
+    items:
+      Warehouse/my-warehouse:
+        commits:
+        - healthCheckCommit: 111eaf55aa41f21bb9bb707ba1baa748b83ec51e
+          id: 961cfaedbc53aacdb65110028839a2c1c281290d
+          repoURL: https://github.com/example/kargo-demo.git
+        images:
+        - digest: sha256:b2487a28589657b318e0d63110056e11564e73b9fd3ec4c4afba5542f9d07d46
+          repoURL: public.ecr.aws/nginx/nginx
+          tag: 1.27.0
+        name: 666209fd9755a1e48bec6b27f5f447747410dd9e
+        origin:
+          kind: Warehouse
+          name: my-warehouse
+    verificationHistory:
+    - analysisRun:
+        name: test.01j2w7aknhf3j7jteyqs72hnbg.101bca5
+        namespace: kargo-demo-09
+        phase: Successful
+      finishTime: "2024-07-15T22:13:57Z"
+      id: 5535a484-bbd0-4f12-8cf4-be2c8e0041c9
+      phase: Successful
+      startTime: "2024-07-15T22:13:34Z"
+  health:
+    argoCDApps:
+    - healthStatus:
+        status: Healthy
+      name: kargo-demo-09-test
+      namespace: argocd
+      syncStatus:
+        revision: 111eaf55aa41f21bb9bb707ba1baa748b83ec51e
+        status: Synced
+    status: Healthy
+  lastPromotion:
+    finishedAt: "2024-07-15T22:13:25Z"
+    freight:
+      commits:
+      - healthCheckCommit: 111eaf55aa41f21bb9bb707ba1baa748b83ec51e
+        id: 961cfaedbc53aacdb65110028839a2c1c281290d
+        repoURL: https://github.com/example/kargo-demo.git
+      name: 666209fd9755a1e48bec6b27f5f447747410dd9e
+      origin:
+        kind: Warehouse
+        name: kargo-demo
+    name: test.01j2w7a15cxjjgejresfyw6ysp.666209f
+    status:
+      finishedAt: "2024-07-15T22:13:25Z"
+      freight:
+        commits:
+        - healthCheckCommit: 111eaf55aa41f21bb9bb707ba1baa748b83ec51e
+          id: 961cfaedbc53aacdb65110028839a2c1c281290d
+          repoURL: https://github.com/example/kargo-demo.git
+        name: 666209fd9755a1e48bec6b27f5f447747410dd9e
+        origin:
+          kind: Warehouse
+          name: kargo-demo
+      freightCollection:
+        id: 101bca5b0e18ca7913978a1da956308d2544f741
+        items:
+          Warehouse/kargo-demo:
+            commits:
+            - healthCheckCommit: 111eaf55aa41f21bb9bb707ba1baa748b83ec51e
+              id: 961cfaedbc53aacdb65110028839a2c1c281290d
+              repoURL: https://github.com/example/kargo-demo.git
+            name: 666209fd9755a1e48bec6b27f5f447747410dd9e
+            origin:
+              kind: Warehouse
+              name: kargo-demo
+        verificationHistory:
+        - analysisRun:
+            name: test.01j2w7aknhf3j7jteyqs72hnbg.101bca5
+            namespace: kargo-demo-09
+            phase: ""
+          id: 5535a484-bbd0-4f12-8cf4-be2c8e0041c9
+          phase: Pending
+          startTime: "2024-07-15T22:13:34Z"
+      phase: Succeeded
+  observedGeneration: 1
+  phase: Steady
+```
+
```

Comment:
Right here seems to be the right place to add something like `## Interacting with Stages`.

You can show how to create, delete, and refresh Stages. Info on promoting _to_ a Stage probably belongs in a not-yet-written "Working with Promotions" page, so I would not include it here (but you can link to it later when it exists). Verifying the Stage's current Freight could also be discussed here. i.e. Refer back to [this comment](https://github.com/akuity/kargo/pull/2608/files#r1844211810) and relocate those details to this new section.

For each of create, delete, refresh, and verify, you can show how these can be accomplished with either the UI or CLI.